locales/en-US/webextension.properties

```diff
@@ -53,3 +53,5 @@ termsAndPrivacyNotice2 = By using Firefox Screenshots, you agree to our {termsAn
 termsAndPrivacyNoticeTermsLink = Terms
 # This string is used as the text for a link in termsAndPrivacyNoticeCloudServices
 termsAndPrivacyNoticyPrivacyLink = Privacy Notice
+# This string is used to label the item in the Library panel that opens the screenshots page
+libraryLabel = Screenshots
```

Comment:
Everywhere else in the product, we use the phrase 'My Shots' to refer to the user's shots page. @johngruen thoughts?